packages/smooth_ui_library/example/pubspec.lock

```diff
@@ -253,7 +253,7 @@ packages:
       name: test_api
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.4.7"
+    version: "0.4.3"
```

Comment:
downgrading from 0.4.7 to 0.4.3?

Reply:
Thats indeed interesting, it's maybe caused by removing the dart test package from the smooth_ui_library (Just using flutter_test instead) but the tests still work so it is likely not a bug problem